apps/configurator/lib/configurator_web/live/game_modes/form.html.heex

```diff
@@ -0,0 +1,82 @@
+<.simple_form :let={f} for={@changeset} action={@action} phx-submit={@action} phx-change="validate">
+  <.error :if={@changeset.action}>
+    Oops, something went wrong! Please check the errors below.
+  </.error>
+  <p class="font-semibold"><%= "Version: #{@version.name}" %></p>
+  <.input type="hidden" field={f[:version_id]} value={@version.id} />
+  <.input
+    field={f[:name]}
+    type="select"
+    label="Game Mode"
+    options={Ecto.Enum.values(GameBackend.CurseOfMirra.GameModeConfiguration, :name)}
+  />
+  <.input field={f[:zone_enabled]} type="checkbox" label="Zone enabled" />
+  <.input field={f[:bots_enabled]} type="checkbox" label="Bots enabled" />
+  <.input field={f[:match_duration_ms]} type="number" label="Match duration ms" />
+  <.input field={f[:respawn_time_ms]} type="number" label="Respawn time ms" />
+
+  <div class="flex items-center justify-between gap-6">
+    <span
+      phx-click="add_map_params"
+      class="relative phx-submit-loading:opacity-75 rounded-lg bg-zinc-900 hover:bg-zinc-700 py-2 px-3 text-sm font-semibold leading-6 text-white active:text-white/80"
+    >
+      Add Map Params
+    </span>
+  </div>
+
+  <div class="flex items-center justify-between gap-6">
```

Comment:
We can add an X button within every nested form, so we can delete an specific configuration and not the last one